client/src/pages/TourPage/TourPage.js

```diff
@@ -147,19 +147,21 @@ const TourPage = () => {
 
   const dispatch = useDispatch();
   // to be revised in future from re rendering and optimizing point of view, whether we pass needed data as props to components or useSelector directly in each component.
-  const { imageUrls } = useSelector((store) => store.tour.data);
+  const { imageUrls, name, description, reasons } = useSelector((store) => store.tour.data);
 
   useEffect(() => {
     dispatch(fetchTour());
   }, []);
 
   return (
     <>
+      <div>
+        <ImageGallery />
+      </div>
       <HeaderContent>
         <Container>
           <Typography align="center" variant="h1" mt={17} mb={5} fontSize="50px">
```

Comment:
Suggest capitalizing first letter of the title

```suggestion
          <Typography
            align="center"
            variant="h1"
            mt={17}
            mb={5}
            fontSize="50px"
            sx={{
              ':first-letter': {
                textTransform: 'capitalize',
              },
            }}
          >
```

Reply:
done